validator/Cargo.toml

```diff
@@ -16,3 +16,7 @@ idna = "0.1"
 serde = "1"
 serde_derive = "1"
 serde_json = "1"
+phonenumber = { version = "0.1.0+8.7.0", optional = true }
```

Comment:
that's some weird version number :o

Reply:
Yes, could not figure out why. That library is a Rust port of [https://github.com/googlei18n/libphonenumber](googlei18n/libphonenumber), for which the `8.x.x` part belongs.